conda/repodata.py

```diff
@@ -0,0 +1,99 @@
+# (c) Continuum Analytics, Inc. / http://continuum.io
```

Comment:
To illustrate what I meant in my last comment in this thread, I think a user using this API should only import modules directly from `conda` to get all the public APIs:
```
from conda.repodata import RepoData
from conda.resolver import Resolver
...
```
Among other reasons, this makes it clear whenever a user is trespassing into non-public APIs, and it avoids having to maintain the mess that are all the numerous "public" modules including `conda.api` and `conda.exports`.